Sources/PT.PM.Common/Nodes/RootUst.cs

```diff
@@ -9,31 +9,29 @@
 namespace PT.PM.Common.Nodes
 {
     [MessagePackObject]
-    [MessagePackFormatter(typeof(RootUstFormatter))]
     public class RootUst : Ust
     {
         private Language[] sublanguges;
 
         [Key(UstFieldOffset)]
-        public Language Language { get; }
+        public Language Language { get; set; }
 
-        [JsonProperty("SourceCodeFile")] // TODO: back compatibility with external serializers
-        [Key(UstFieldOffset + 1)]
+        [JsonProperty("SourceCodeFile"), JsonIgnore, IgnoreMember] // TODO: back compatibility with external serializers
```

Comment:
What's the point in keeping `JsonProperty` attribute with `JsonIgnore`?

Reply:
It's a workaround for external jsons deserialization. I'll mark it with a comment.